niftynet/io/misc_io.py

```diff
@@ -779,10 +783,14 @@ def close_logger():
 
     :return:
     """
-    # pylint: disable=no-name-in-module
-    from tensorflow.python.platform.tf_logging import _get_logger
+    try:
```

Comment:
I would comment here that this is needed for compatibility reasons between TF versions